### PR TITLE
react-form-controls | Rich Trunzo 

### PR DESCRIPTION
**Pull request by richtrunzo**
Screen Shot 2021-01-20 at 3 16 24 PM

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+  handleSubmit(){
+    event.preventDefault();
+    console.log('state:', this.state);
+  }
+  render(){
+    return <form onSubmit={this.handleSubmit}>
+            <label>
+              Email:
+              <input type="email" onChange={this.handleChange}></input>
```
Comment (scbowler) [inline react-form-controls/src/index.jsx]: Here you need to have a `value` attribute set to `this.state.value`

File: react-form-controls/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    this.state = {value: " "};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleChange(){
+    this.setState({value: event.target.value});
+  }
+  handleSubmit(){
+    event.preventDefault();
+    console.log('state:', this.state);
```
Comment (scbowler) [inline react-form-controls/src/index.jsx:16]: Here you should be updating the `value` property in state with the value of the input, HINT you need a parameter named event

**Review by scbowler — CHANGES_REQUESTED** [review]
React needs to control the input so you should be using state to set the value of the input.

**Review by BrettAlbright — APPROVED** [review]
👍

**Pull request merged**